patches/common/v8/do_not_export_private_v8_symbols_on_windows.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tomas Rycl <torycl@microsoft.com>
+Date: Mon, 13 May 2019 15:48:48 +0200
+Subject: Do not export private V8 symbols on Windows
+
+
```

Comment:
NB. we require that every patch contains an explanation of why it is necessary and can't be implemented without needing a patch. Something like:

> Since Electron forces BUILDING_V8_SHARED, but doesn't need to export V8's private symbols (since we don't run V8's tests in component mode), we prevent private symbols from being exported here. This reduces the number of symbols that electron.lib exports and consequently [DESCRIBE VALUE OF REDUCING SYMBOL COUNT]

Also maybe consider merging this with the other patch for BUILDING_V8_SHARED?

Reply:
Thanks for comment. I am now in the middle of testing. I still don't know if this is the best approach. I definitely need to reduce number of symbols and remove stl ones. But I don't know if this is the right way to do this. That is the reason this is just a draft. I needed to see what will happen if I try to disable private V8 symbols.

Reply:
Does reducing the number of symbols have any performance effect?

Reply:
Performance isn't the only potential reason for this PR, but @torycl I agree with @nornagon that it would be great if you'd update the PR description to discuss the problem you're trying to solve -- e.g. does exporting stl symbols cause conflicts somewhere?

Reply:
Yes. It causes crashes of node modules which just use std::basic_ostream and dll crt on Windows. I am trying to fix this by filtering number of symbols which will make it to final node.lib.